packages/dashboard/src/customization/propertiesSections/settings/section.tsx

```diff
@@ -43,17 +59,45 @@ export const SettingsSection = ({
             className='settings-property-label'
             style={{ gap: awsui.spaceScaledS }}
           >
-            <label htmlFor='decimal-places'>Decimal places</label>
-            <div className='settings-property-label-control'>
-              <Input
-                type='number'
-                data-testid='decimal-place-config'
-                controlId='decimal-places'
-                ariaLabel='decimal places'
-                value={significantDigits?.toFixed() ?? ''}
-                onChange={onSignificantDigitsChange}
-              />
-            </div>
+            <label htmlFor='decimal-places'>
+              <b>Places</b>
+            </label>
```

Comment:
This label does not match the styling of the other labels in the config panel. All the others are above the input. This label is to the left of the input. Can we please change that?

Reply:
I see this is what the mock is showing in the issue. I'll need to talk with Navita and follow-up on this.